.github/workflows/license-checker.yml

```diff
@@ -3,10 +3,10 @@ name: License checker
 on:
   push:
     branches:
-      - master
+      - distribute-reorg
   pull_request:
     branches:
-      - master
+      - distribute-reorg
```

Comment:
Sorry. Recently, someone raised the issue of the naming specification of the feature branch, which I changed its name to feature/distribute-reorg.